res/controllers/Behringer-Extension-scripts.js

```diff
@@ -230,21 +273,21 @@
      * @public
      * @see https://github.com/mixxxdj/mixxx/wiki/Script-Timers
      */
-    var Timer = function(options) {
-        _.assign(this, options);
+    const Timer = function(options) {
+        Object.assign(this, options);
         this.disable();
     };
     Timer.prototype = {
         disable: function() { this.id = 0; },
         isEnabled: function() { return this.id !== 0; },
         start: function() {
             this.reset();
-            this.id = engine.beginTimer(this.timeout, function() {
+            this.id = engine.beginTimer(this.timeout, () => {
                 if (this.oneShot) {
                     this.disable();
                 }
                 this.action.call(this.owner);
-            }.bind(this), this.oneShot); // .bind(this) is required instead of arrow function for Qt < 6.2.4 due to QTBUG-95677
+            }, this.oneShot);
```

Comment:
2.5 is still technically supposed to be Qt5 compatible, I think we can't remove this yet.

Reply:
OK, reverted.

Reply:
Would it be OK to apply this removal to 2.6 in a separate PR, @Swiftb0y?

Reply:
Yeah, `main`/`2.6` is supposed to be Qt6, so you can remove that in that.